src/encryption.rs

```diff
@@ -151,6 +151,14 @@ impl<G: Group> Ciphertext<G> {
         G::serialize_element(&self.blinded_element, &mut bytes[G::ELEMENT_SIZE..]);
         bytes
     }
+
+    pub fn new(random_element: G::Element, blinded_element: G::Element) -> Self {
```

Comment:
Again, `from_parts` looks like a better name for this method. Also, it should have docs.